### DISCUSSION TITLE
'pct_reads_in_peaks' and 'a fraction of reads in the peak' by FRiP

### DISCUSSION THREAD

**Discussion by akhst7** [stuart-lab/signac#1351] [General]
Hi, 

FRiP generates a 'fraction of reads in the peak',  times 100 of which should be  equivalent to 'pct_reads_in_peaks' described under  Computing QC Metrics in Analyzing PBMC scATAC-seq or am I wrong on this ? 
 
I used pbmc_granulocyte_sorted_3k_atac_fragments.tsv.gz from 10x and generated a total.fragment counts by following;

`total.fragments<-CountFragments("~/Downloads/fragments/pbmc_granulocyte_sorted_3k_atac_fragments.tsv.gz")`
 
From there, I used FRiP to get the 'fraction of reads in the peak' as follows,

```
pbmc <- FRiP(
object = pbmc,
assay = 'ATAC',
total.fragments = 'total.fragments'
)
```
A summary of the resulting 'fraction of reads in the peak' is following;

```
summary(pbmc$FRiP)
    Min.  1st Qu.   Median     Mean  3rd Qu.     Max. 
  0.0025   0.8414   1.2725   3.8284   1.8448 378.7273 

```
As you can see, a distribution looks ZINB, and if I use the QC criterium regarding  'pct_reads_in_peaks' (Cells with low values (i.e. <15-20%)) in 'Analyzing PBMC scATAC-seq' vignette, there are very few cells left to play with.   

Looking at the FRiP function,  

`frip <- peak.counts/total_fragments_cell`

FRiP generate a fraction of peaks (where read pileups are)  in fragments per a single cell rather than a fraction of reads in fragments per cell.   

Am I missing anything here ?  Any pointer will be appreciated.

**Comment by akhst7** [comment]
Ok I figures this out.

**Reply by Susan920715** [reply]
Hi, Can you tell me whether FRiP generates a 'fraction of reads in the peak', times 100 of which should be equivalent to 'pct_reads_in_peaks' .I have the same confusions, thankyou verymuch

**Reply by akhst7** [reply]
@Susan920715 , `FRiP` will give you the 'fraction of reads in the peak' if you feed it the **right** `peak.count` and `total_fragments_cell`.  The problem, not explained well in the vignette is that very definitions of `total peak, read, and fragment counts` differ if you are dealing with 10X data analyzed  by CellRanger ARC or ATAC.  In my case,  I thought that the read and fragment counts (vs cut-site counts) generated by CR-ARC/ATAC were the same but in fact they were different. For CR-ARC/ATAC, you can take a look at definitions of these in the Summary Metrics page, as 10x changed summary metadata (again!!).   If you can, you should calculate this manually with using a **right**  set of` peak and total frag counts `   I think using `cut-site counts` rather than `fragment counts` in  the `peaks `  may be more suitable describing the number of Tn5 accessible open chromatin.

**Reply by Susan920715** [reply]
I see ~ Thank you very much!!
